### ISSUE TITLE
helloo 

### ISSUE THREAD

**Issue by 4rl3k1n6** [fawazahmed0/Moodle-Hacking#1]
hello im a teacher i wanna know how you made that possible

**Comment by fawazahmed0** [comment]
I have not exploited any vulnerability in moodle, rather it is same as cheating from your notebook in exam.
It a client-server program,
Client:
Client (Student) side program will send the questions to server.
Server:
Server side program will receive the questions and search the answers using elastic search, and send back the answer to the Client.

Elastic Search will have all the subject related OCR textbooks(Searchable)

In simple terms, it is same as someone searching the answers from a textbook to cheat in exam, but my program automatically searches the answer from the OCR textbook and sends back to the user.


________________________________

**Comment by 4rl3k1n6** [comment]
can i download the program?

**Comment by fawazahmed0** [comment]
Hi,
 I can't give the code, it will be misused. The code has to be injected through the browser, also you have to setup a server, to get answers back.

________________________________

**Comment by 4rl3k1n6** [comment]
oh boy..please give me the instructions

sorry for  any inconvenience

**Comment by fawazahmed0** [comment]
Right click in browser -> inspect element ->  console Tab -> This is where my code has to be pasted/injected

________________________________

**Comment by 4rl3k1n6** [comment]
beautifull can i have the code? thank you!

**Comment by 4rl3k1n6** [comment]
and sorry for any problem

**Comment by 4rl3k1n6** [comment]
Hello please send me the code

Best regards.
Pablo
Alea jacta est.

**Comment by fawazahmed0** [comment]
Sorry, I can't , but you can write the code yourself, learn javascript,php. That's what I used in the code.

Regards,
Fawaz Ahmed

________________________________

**Comment by Exelor277** [comment]
hello
I know you didnt want to give the code for ethical reasons but can you make an exception please I've been trying to recreate it but I just can't... I am litteraly stuck at this point
